.github/workflows/internal.yml

```diff
@@ -0,0 +1,33 @@
+name: Internal-unit-tests
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1
+    - name: Build Internal
+      working-directory: internal
+      run: go build ./...
+
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1
+    - name: Run Internal Unit Tests
+      working-directory: internal
+      run: go test -v ./...
```

Comment:
💡 it might be nice to add coverage with `-cover` in the future. I don't know if we'd like to keep a certain code coverage percentage in the future, but could help the reviewer see how well tested the code is.

Reply:
ooh TIL about `-cover` 🙌 . I think that's a good idea added
https://github.com/stellar/stellar-etl/pull/130/commits/d27f7fb8786f7bc758f7b447307aebc4d18938e3